README.md

```diff
@@ -76,6 +62,32 @@ Create a profile, review the sample use cases to familiarize yourself with the c
 
 You can now use all the functionalities of the extension.
 
+### Use Base Profiles
+
+Leverage existing base profiles with a token to log in to API Mediation Layer.
+
+Before using the base profiles functionality, ensure that you have [Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html) installed.
+
+**Follow these steps:**
+
+1. Open Zowe CLI and run the following command:
+
+   ```
+   zowe auth login apiml
+   ```
+
+2. Follow the instructions and provide all necessary information to create a base profile. 
+
+   You have successfully created a base profile.
+  
+3. Run Zowe Explorer and click the **+** icon.  
+
+4. Select the z/OSMF profile you use with your base profile.  
```

Comment:
Select the profile that you use with the token created for your local base profile.

Reply:
yeah, it doesn't have to be a zosmf profile.
Good catch!

Reply:
so there can be a base profile without a token?